quizzes/midterm.md

```diff
@@ -90,23 +128,40 @@
 	my_hash = { "a" => 100, "b" => 200, "c" => 300 }
 	my_hash.select {|key,value| key < "c"}
 	```
+	
+	```
+	{ nil }
+	```
 
 1. Write code that iterates through the numbers array and out`puts` the `KEY` for `each` of the values in the array.
 
 	```ruby
 	numbers = ['one', 'two', 'three']
 	```
+	
+	```
+	def init
+	  puts "numbers"
+	end
+	```
 
 1. Write code that `maps` the fruits array to a new array where all the values are `reverse`d. Eg. "apples" = "selppa"
```

Comment:
10/10. How did you get the above question wrong but this right? They are almost identical. :confused: =)